### PR TITLE
Removing redundant libraries

### PR DESCRIPTION
**Pull request by delta-G**
I am purposefully removing redundant libraries that are not needed anymore.  These will be removed from GitHub shortly

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @delta-G. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

**Review by MatteoPologruto — APPROVED** [review]
Done:
```
2024/12/11 16:54:38 Loaded 7567 libraries from DB
Removing EventLinkInterrupt
warning: Library Git clone folder /home/ubuntu/workspace/gitclones/github.com/delta-G/EventLinkInterrupt not present
Success!
```
```
2024/12/11 16:55:21 Loaded 7566 libraries from DB
Removing GPT_Stepper
Success!
```
```
2024/12/11 16:56:10 Loaded 7565 libraries from DB
Removing LoveButton
Success!
```
```
2024/12/11 16:57:11 Loaded 7564 libraries from DB
Removing PID_DG
Success!
```
```
2024/12/11 16:58:00 Loaded 7563 libraries from DB
Removing R4_Touch
Success!
```

**Pull request merged**
